modules/distr-tracing-tempo-troubleshooting-issues-with-ingesting-traces.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="problems-ingesting-traces_{context}"]
+= Troubleshooting issues with ingesting traces
+
+When the TempoStack instance is failing to ingest traces, you can troubleshoot it as follows.
+
+.Procedure
+
+. Review the components in the ingestion path to diagnose the issue:
++
+* The spans are not sent correctly.
+* The spans are not sampled.
+
+. Check for errors in the logs that identify unhealthy ingesters, which might be failing due to out-of-memory (OOM) issues or scale-down events.
++
+In high-volume tracing environments, the default trace limits might not be adequate.
+If spans are being refused due to these limits, you can discover this in the distributor logs.
++
+You can also see that the `tempo_discarded_spans_total` metric is incremented.
+
+. Check the following two metrics to verify that the TempoStack instance is successfully receiving traces:
++
+[source,yaml]
+----
+tempo_distributor_spans_received_total
+tempo_ingester_traces_created_total
+----
```

Comment:
Same issues as previous module - it's not clear how a user would check these, if there are specific places they should look or commands they should run?

Reply:
@rubenvp8510, WDYT?

Reply:
I'd say that these are sufficient directions for the target audience that is able to use this product.